eclipse/che-operator
# fix: Fix scripts and che-types to use newer operator-sdk

### What does this pull request do
Fix applying csv samples to the csv files.

## Referenced issue:
https://github.com/eclipse/che/issues/18349

### How to test it:
Download and install operator-sdk 0.17.1
1) Generate new bundles:

```bash
$ ./olm/update-nightly-bundle.sh
```

Bundle files should be updated.

2) Launch some cluster, login to cluster, launch debugger:

```bash
./local-debug.sh
```

3) Update crds:

```bash
$ ./olm/update-crd-files.sh
```

Launch VSCode task `Che Operator`.

Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>
